flatten_epub_toc.py

```
@@ -77,6 +80,28 @@ def flatten_nav(data, title_label, toc_label):
+    root = ET.fromstring(data)
+    namespace = root.tag.partition("}")[0].lstrip("{")
+    spine_tag = f"{{{namespace}}}spine" if namespace else "spine"
+    spine = root.find(spine_tag)
+    if spine is None:
+        raise RuntimeError("EPUB package spine not found")
+    spine.set("page-progression-direction", "rtl")
+    return ET.tostring(root, encoding="utf-8", xml_declaration=True)
+
+
+def set_xhtml_direction(data):
+    root = ET.fromstring(data)
```

Ruff (0.15.21)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)